packages/nl-design-system-unstable/status-badge-component/README.md

```diff
@@ -0,0 +1,79 @@
+# Status badge
+
+*Status: **In development***
+
+## Introduction
+
+A status badge draws attention and alerts a user to system states. The information in a status badge is short, concise and clearly scannable.
+
+## Variants
+
+- Success
+- Warning
+- Error
+
+## Alternatives and related components
+
+- Status light
+- Text badge
+
+## Anatomy
+
+- Container with padding
+- Status text
+
+## Design tokens
+- --nl-status-badge-border-radius // Border radius
```

Comment:
Since the design tokens are formatted as CSS variables, I think they would look better as code. You can put backticks around them for this. For example:

- `--nl-status-badge-border-radius`: Border radius

Reply:
Fixed, will update the contribution documentation to make sure that tokens are always displayed as code snippet.